src/Nethermind/Nethermind.Network/P2P/Subprotocols/Eth/V63/Messages/ReceiptsMessageSerializer.cs

```diff
@@ -40,7 +40,16 @@ public void Serialize(IByteBuffer byteBuffer, ReceiptsMessage message)
 
         public ReceiptsMessage Deserialize(IByteBuffer byteBuffer)
         {
-            if (byteBuffer.Array.Length == 0 || byteBuffer.Array.First() == Rlp.OfEmptySequence[0]) return new ReceiptsMessage(null);
+            if (byteBuffer.ReadableBytes == 0)
+            {
+                return new ReceiptsMessage(null);
```

Comment:
Wouldn't it be better to have some `ReceiptsMessage.Empty` ?

Reply:
Sure enough.